cmd/kueuectl/app/stop/stop_clusterqueue.go

```diff
@@ -83,7 +87,7 @@ func NewClusterQueueCmd(clientGetter util.ClientGetter, streams genericiooptions
 }
 
 // Complete completes all the required options
-func (o *ClusterQueueOptions) Complete(clientGetter util.ClientGetter, cmd *cobra.Command, args []string) error {
+func (o *ClusterQueueOptions) Complete(clientGetter util.ClientGetter, args []string) error {
```

Comment:
Maybe this should be done in another cleanup.

Reply:
Reverted

Reply:
Created follow up #2591.